python/demo.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: UTF-8 -*-
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from pytorch_transformers import BertTokenizer, BertForTokenClassification, BertConfig
+import streamlit as st
+import torch
+import yaml
+
+# Run using `streamlit run demo.py`
+
+# NOTE: Change CHK_PATH to your preferred model checkpoint path before running.
+# NOTE 2: Argparse currently breaks the demo if you try to pass args in the command
+# line. If you'd like to use the demo in Russian, change argparse_default_lang to 'ru'
+# below.
+#
+# Example path:
+# CHK_PATH = "../models/10.03.2019, 17.12.59/train_checkpoint_epoch_5.tar"
+CHK_PATH = "YOUR_CHECKPOINT_PATH_HERE"
+ARGPARSE_DEFAULT_LANG = "en"
+
+
+@st.cache()
+def load_model_and_tokenizer(chk_path, state_dict_name="model_state_dict", lang="en"):
+
+    """
+    Loads model from a specified checkpoint path. Replace `CHK_PATH` at the top of
+    the script with where you are keeping the saved checkpoint.
+    (Checkpoint must include a model state_dict, which by default is specified as
+    'model_state_dict,' as it is in the `main.py` script.)
+    """
+
+    model_name = "bert-base-cased" if lang == "en" else "bert-base-multilingual-cased"
+
+    checkpoint = torch.load(chk_path)
+    model_state_dict = checkpoint[state_dict_name]
+    model = BertForTokenClassification.from_pretrained(
+        model_name, num_labels=NUM_LABELS
+    )
+    model.load_state_dict(model_state_dict)
+    tokenizer = BertTokenizer.from_pretrained(model_name, do_lower_case=False)
+
+    return model, tokenizer
+
+
+def predict_on_text(input_text):
```

Comment:
these functions should be imported into a file that has just the if __name__ == "__main__" statement below